src/Draco.Compiler.Tests/TestUtilities.cs

```diff
@@ -8,21 +8,24 @@ namespace Draco.Compiler.Tests;
 
 internal static class TestUtilities
 {
+    public const string DefaultAssemblyName = "Test.dll";
     public static string ToPath(params string[] parts) => Path.GetFullPath(Path.Combine(parts));
 
-    public static MetadataReference CompileCSharpToMetadataRef(string code)
+    public static MetadataReference CompileCSharpToMetadataRef(string code, string assemblyName = DefaultAssemblyName, IEnumerable<Stream>? aditionalReferences = null)
     {
-        var stream = CompileCSharpToStream("Test.dll", code);
+        var stream = CompileCSharpToStream(code, assemblyName, aditionalReferences);
         return MetadataReference.FromPeStream(stream);
     }
 
-    public static Stream CompileCSharpToStream(string assemblyName, string code)
+    public static Stream CompileCSharpToStream(string code, string assemblyName, IEnumerable<Stream>? aditionalReferences = null)
```

Comment:
You could also default assemblyName here and get rid of the bunch of reordered parameters